public/components/trace_analytics/components/common/plots/service_map.tsx

```diff
@@ -443,35 +407,57 @@ export function ServiceMap({
             <EuiFlexItem grow={7}>
               <EuiPopover
                 button={
-                  <EuiFieldSearch
-                    compressed
-                    prepend="Focus on"
-                    placeholder="Service name"
-                    value={focusedService || query}
-                    onClick={() => setPopoverOpen(!isPopoverOpen)}
-                    onChange={(e) => {
-                      const newValue = e.target.value;
-                      setQuery(newValue);
-                      if (newValue === '') {
-                        setGraphKey((prevKey) => prevKey + 1);
-                        setQuery('');
-                        onFocus('');
-                      }
-                    }}
-                    isInvalid={query.length > 0 && invalid}
-                    append={
-                      <EuiButtonIcon
-                        iconType="refresh"
-                        size="s"
-                        onClick={() => {
-                          setGraphKey((prevKey) => prevKey + 1);
-                          setQuery('');
-                          onFocus('');
-                        }}
-                      />
+                  <EuiToolTip
+                    position="top"
+                    content={
+                      isFocusBarDisabled
+                        ? 'To use the focus field, clear the service filter manually or click the refresh icon.'
+                        : undefined
                     }
-                    aria-controls="service-select-dropdown"
-                  />
+                  >
+                    <EuiFieldSearch
+                      compressed
+                      prepend="Focus on"
+                      placeholder={focusedService || 'Service name'}
+                      value={query}
+                      onClick={() => {
+                        if (!isFocusBarDisabled) setPopoverOpen(!isPopoverOpen);
+                      }}
+                      onChange={(e) => {
+                        if (!isFocusBarDisabled) {
+                          const newValue = e.target.value;
+                          setQuery(newValue);
+                          if (newValue === '') {
+                            if (focusedService) {
+                              setGraphKey((prevKey) => prevKey + 1);
+                              setQuery('');
+                              onFocus(focusedService);
+                            } else {
+                              setGraphKey((prevKey) => prevKey + 1);
+                              setQuery('');
+                              onFocus('');
+                            }
+                          }
```

Comment:
May be move repeated line outside this `if`:

```suggestion
                          if (newValue === '') {
                           setGraphKey((prevKey) => prevKey + 1);
                           setQuery('');
                           focusedService ?  onFocus(focusedService) : onFocus('');
                          }
```

Reply:
Changed too     
if (newValue === '') {
                            setGraphKey((prevKey) => prevKey + 1);
                            setQuery('');
                            onFocus(focusedService || '');
                          }

Reply:
Thanks!